pkg/scheduler/plugins/topology/job_filtering.go

```diff
@@ -17,10 +17,10 @@ import (
 	"github.com/NVIDIA/KAI-scheduler/pkg/scheduler/log"
 )
 
-type domainsByLevel map[string]map[TopologyDomainID]*TopologyDomainInfo
+type domainsByLevel map[string]map[DomainID]*DomainInfo
```

Comment:
Maybe it will be nice to create a dedicated type for the domain level as well?
Something like:
```
type domainsByLevel map[DomainLevel]map[DomainID]*DomainInfo
```

Reply:
Done

Reply:
Ok, looks good to me